src/models/workspaces.ts

```diff
@@ -105,36 +105,61 @@ export const WorkspacesModel = types
     learningLogs: types.array(LearningLogWorkspaceModel),
     publications: types.array(PublishedWorkspaceModel)
   })
-  .actions((self) => {
+  .views((self) => {
```

Comment:
There were a bunch of getters defined in `actions`, which I moved into `views`. This was tripping me up for a while - I originally defined `getLatestPublicationsForSection` as an `action`, which is why I was having trouble getting the "Class Work" tab to re-render

Reply:
That would do it. 😉